setup.py

```diff
@@ -23,9 +23,21 @@ def read(fname):
     description="Common fixtures for inmanta LSM related modules",
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    packages=["pytest_inmanta_lsm"],
     package_dir={"": "src"},
-    install_requires=["pytest-inmanta", "inmanta-lsm"],
+    package_data={
+        "pytest_inmanta_lsm": [
+            "resources/docker-compose.yml",
+            "resources/my-env-file",
+            "resources/my-server-conf.cfg",
+            "py.typed",
+        ]
+    },
+    include_package_data=True,
+    install_requires=[
+        "pytest-inmanta~=2.3",
+        "docker-compose~=1.29",
```

Comment:
I wonder if we should make this optional or always just pull in the whole docker-python toolchain?

Reply:
Not sure, is there a clean way of doing this?

Reply:
We could probably add it in another pytest extension (pytest-inmanta-lsm-ctr) but that would makes things complicated no?